tests/integration/security/rbac/grouplist/testdata/istio-group-list-rbac-rules.yaml.tmpl

```diff
@@ -0,0 +1,97 @@
+# istio-group-list-rbac-rules.yaml to enforce access control based on
+# groups and list claims for both http and tcp services using Istio RBAC rules.
+
+# For service a, allow no one to access it.
```

Comment:
Is this needed? I don't see any test case for service a in `rbac_group_list_test.go`

Reply:
Done: this is not needed and has been removed.